pkg/logs/input/file/parser.go

```diff
@@ -0,0 +1,109 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package file
+
+import (
+	"bytes"
+	"errors"
+
+	"github.com/DataDog/datadog-agent/pkg/logs/message"
+	logParser "github.com/DataDog/datadog-agent/pkg/logs/parser"
+)
+
+const (
+	// Containerd log partial flags
+	logTagPartial = 'P'
+	logTagFull    = 'F'
+	// Containerd log stream type
+	stdout = "stdout"
+	stderr = "stderr"
+)
+
+// containerdFileParser parses containerd file logs
+var containerdFileParser *parser
+
+type parser struct {
+	logParser.Parser
+}
+
+// Parse parse log lines of containerd
+// These line have the following format
+// Timestamp ouputchannel partial_flag msg
+// Example:
+// 2018-09-20T11:54:11.753589172Z stdout F This is my message
+func (p *parser) Parse(msg []byte) (*message.Message, error) {
+	// timestamp goes till first space
+	timestamp := bytes.Index(msg, []byte{' '})
```

Comment:
Would be nice to mimic what's done in `docker/parser` https://github.com/DataDog/datadog-agent/blob/1e5ef8239681daf319cd834897dcf2ddaa380b45/pkg/logs/input/docker/parser.go#L35